api/queries_pr.go

```diff
@@ -531,67 +531,23 @@ func pullRequestFragment(httpClient *http.Client, hostname string) (string, erro
 		return "", err
 	}
 
-	var reviewsFragment string
-	if prFeatures.HasReviewDecision {
-		reviewsFragment = "reviewDecision"
+	fields := []string{
+		"number", "title", "state", "url", "isDraft", "isCrossRepository",
+		"headRefName", "headRepositoryOwner", "mergeStateStatus",
 	}
-
-	var statusesFragment string
 	if prFeatures.HasStatusCheckRollup {
-		statusesFragment = `
-		commits(last: 1) {
-			nodes {
-				commit {
-					statusCheckRollup {
-						contexts(last: 100) {
-							nodes {
-								...on StatusContext {
-									state
-								}
-								...on CheckRun {
-									conclusion
-									status
-								}
-							}
-						}
-					}
-				}
-			}
-		}
-		`
+		fields = append(fields, "statusCheckRollup")
 	}
 
-	var requiresStrictStatusChecks string
-	if prFeatures.HasBranchProtectionRule {
-		requiresStrictStatusChecks = `
-		baseRef {
-			branchProtectionRule {
-				requiresStrictStatusChecks
```

Comment:
Was this meant to be deleted? Looks like it is still in use in `pr status`.

Reply:
Yikes; good catch! I need to add this back to the overall query.